dll/win32/newdev/wizard.c

```diff
@@ -641,9 +641,13 @@ CHSourceDlgProc(
                 {
                     BROWSEINFO bi = { 0 };
                     LPITEMIDLIST pidl;
+                    HINSTANCE hInst = (HINSTANCE)GetWindowLongPtr(hwndDlg, GWLP_HINSTANCE);
+                    WCHAR Title[MAX_PATH];
+                    LoadStringW(hInst, IDS_BROWSE_FOR_FOLDER_TITLE, Title, _countof(Title));
```

Comment:
Based on @HBelusca suggestion:
```suggestion
                    WCHAR Title[MAX_PATH];
                    LoadStringW(hDllInstance, IDS_BROWSE_FOR_FOLDER_TITLE, Title, _countof(Title));
```